scala/private/phases/api.bzl

```diff
@@ -42,7 +42,7 @@ def _adjust_phases(phases, adjustments):
     return phases
 
 # Execute phases
-def run_phases(ctx, builtin_customizable_phases):
+def run_phases(ctx, builtin_customizable_phases, target = None):
```

Comment:
Maybe add a comment that `target` is given only in aspects context

Reply:
I'll do this, I think it's more clear:
```
def run_phases(ctx, builtin_customizable_phases):
    return _run_phases(ctx, builtin_customizable_phases, target = None)

def run_aspect_phases(ctx, builtin_customizable_phases, target):
    return _run_phases(ctx, builtin_customizable_phases, target)

def _run_phases(ctx, builtin_customizable_phases, target):
    ...
```